client/elements/sc-site-layout.js

```diff
@@ -147,25 +147,16 @@ export class SCSiteLayout extends LitLocalized(LitElement) {
       this.actions.changeDisplaySuttaInfoState(false);
     });
 
-    let scrollDistance = 0;
+    // handles scroll-based universal toolbar animation on home page
     document.addEventListener(
       'scroll',
-      throttle(500, () => {
-        if (!this.toolbarPosition.scrollForToolbar) {
+      rafThrottle(() => {
+        if (!this.toolbarPosition.scrollForToolbar || this.changedRoute.path !== '/') {
           return;
         }
-        const syntheticEvent = new WheelEvent('syntheticWheel', { deltaY: 4, deltaMode: 0 });
```

Comment:
I am not 100% certain what this is meant to do but it seems to be a secondary way of rate limiting? Please let me know if there was some other purpose.

Reply:
Thanks for the improvement ：）